README.md

```diff
@@ -165,31 +112,34 @@ Can be combined with `eo:bands`'s [`Band Object`][stac-ext-eo-band-obj].
 | tag         | string  | Tag of the image.                                     |
 | working_dir | string  | Working directory in the instance that can be mapped. |
 | run         | string  | Running command.                                      |
-| gpu         | boolean | True if the docker image requires a GPU.              |
+| accelerator         | boolean | True if the docker image requires a custom accelerator (CPU,TPU,MPS).              |
+
+### Output
 
-### Outputs Object
+| Field Name               | Type                    | Description                                                                                                                                                                        |
+|--------------------------|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| task                     | [Task Enum](#task-enum) | Specifies the Machine Learning task for which the output can be used for.                                                                                                          |
+| number_of_classes        | integer                 | Number of classes.                                                                                                                                                                 |
+| output_shape        | \[integer]              | Shape of the output array/tensor from the model For example ($N \times H \times W$). Use -1 to indicate variable dimensions, like the batch dimension.                                                                                                                   |
+| class_name_mapping       | dict                    | Mapping of the output index to a short class name, for each record we specify the index and the class name.                                                                        |
```

Comment:
This inherits from Python, but does not translate properly to JSON (cannot have integer indices). 
We should define a proper "Mapping" object (instead of `dict`) and adjust the definition to use 
`{ "<class-name>": <index> }`, or be even more explicit as Bands do with an array of `{"name": "...", index: <int> }`.

Reply:
will do, and the description is reversed, I'm actually using string: index mapping currently.

Reply:
I'm currently leaning toward the first option, describing this such that each "<class_name>" is it's own field name and it's type is an "<index>" with type integer.

This would mean the schema for the extension would have a flexible number of fields for this object. In practice it looks like this can be implemented like so in JSON

```
{
    "type": "object",
    "patternProperties": {
        "^[a-zA-Z][0-9]+$": { "type": "string" }
    },
    "additionalProperties": {
        "type": ["integer"]
    }
}

```

where properties refers to the keys and patternProperties enforces that class names start with a letter. additionalProperties refers to the values

I prefer this flexible object because it is less verbose and easier to parse a single dict than an array of many dicts that have standard "name" and "index" fields.

Reply:
Do you have an example of what you have in mind? 
I think a presentation like so is sufficient: 
```json
{
  "background": 0,
  "tree":  1, 
  "car": 2
}
```

I would avoid enforcing a specific pattern to the labels because one could have a specific labeling structure to form pseudo-categories, such as: 
```json
{
  "background": 0,
  "1-tree_spruce":  1, 
  "1-tree_oak":  1, 
  "2-vehicle_car": 2,
  "2-vehicle_truck": 2
}
```

Reply:
yes that's what I had in mind

```
{
  "background": 0,
  "tree":  1, 
  "car": 2
}
```

sure I'll avoid any constraints on the labels other than type.

Reply:
This exists: https://github.com/stac-extensions/classification#classification-types 
It was recommended during the STAC community meeting.
We should reuse the defined extension that is identical to the proposed structure above.